### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by dskw** [WesCook/Homestead#46]
Hey there,

so after some playtime and nearing the industrial age I have a list of possible improvements that would benefit the experience:

- Add a nutrition iconbar to the HUD showing one icon per nutrition-type and maybe with a fillstate, or just an indicator if the player is low on one of types and then show the specific indicator
- Not sure if this is possible, but: Add a filter option to the Kitchen UI to filter by nutrition type
- Also not sure if possible: Questbook items should show the recipe upon clicking a craftable goal

In reference to the animania performance problem: [reddit](https://www.reddit.com/r/feedthebeast/comments/6nt129/announcing_homestead_a_new_tfcinspired_modpack/dmzfjx2/)
Animania was updated to 1.3.2 which improves performance and adds a lot new content.
_But be aware:_ if you update be aware that they changed the milking mechanic:
**Cows only give milk if they have a calf**! (same might apply to goats)
One way to compensate that change and also give the player the RPG option to play a vegetarian style: Make soya milk an alternative for some dairy products :) (I actualy did not check if this is already the case)

So far the rest looks very good balanced and thanks again for the awesome pack :)

**Comment by WesCook** [comment]
Sorry for the delayed response on this.  I wasn't expecting to put off this update for so long, but I've been busy on other projects.

The Animania update will require reworking some things, but I'm glad it's finally out.  I'd been looking forward to goats being added (and finally my quest "Goats in Boats" will actually make sense).

I'll address your other suggestions once I have a day to set aside to work on this again.  Hopefully it won't take another two weeks for that.

Cheers.

**Comment by dskw** [comment]
Hey, no problem.
In the meantime I've also looked into Refined Relocation 2 and tried to add it to the pack. The chests show up fine and work as expected but I do not see the fast hopper which should be in the mod. Could this be caused by some still existing config in the pack?

My reason is that the available hoppers are way to slow to empty a Steves Carts CargoManager and I could not find a good mod that provides a simple solution to that yet.

**Comment by WesCook** [comment]
Just a heads up, I removed Refined Relocation due to a serious item loss bug (#14).  I see there's been a new update to the mod since then however, so hopefully this issue has been resolved.  Please let me know if you run into any issues, and I may consider re-adding the mod.

I'm afraid I'm not sure about fast hoppers off-hand.  They're not in the [JEI blacklist](https://github.com/WesCook/Homestead/blob/master/config/JEI/itemBlacklist.cfg), so they've not been hidden.

**Comment by WesCook** [comment]
I'll be pushing an update soon that includes the new Animania update.  I've updated the quests to explain this, so thanks for the heads up.

To address your other points, unfortunately I'm pretty limited on what I can do.

>Add a nutrition iconbar to the HUD showing one icon per nutrition-type and maybe with a fillstate, or just an indicator if the player is low on one of types and then show the specific indicator

While this would be possible (I'm also the author of Nutrition), I don't currently have the time to implement such a feature.  I've also switched development over to the 1.12 build, so the 1.10 and 1.11 branches are (ironically) no longer supported.

>Not sure if this is possible, but: Add a filter option to the Kitchen UI to filter by nutrition type

Afraid this is required of the respective mod author.  I can't add such a feature without forking their mod.

Of course I'd love if they were willing to support my mod.  :)

>Also not sure if possible: Questbook items should show the recipe upon clicking a craftable goal

Afraid that's not possible either.  BetterQuesting is going through (another) rewrite, and this may be possible once v3 is released.  Though we're probably looking at a new version of Minecraft for that.

I realize I also mentioned the Refined Relocation issue to you over on reddit, so sorry for the duplicate info.
